app/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+var yosay = require('yosay');
+var mkdirp = require('mkdirp');
+
+
+module.exports = yeoman.generators.Base.extend({
+  prompting: function () {
+    var done = this.async();
+
+    // Have Yeoman greet the user.
+    this.log(yosay(
+      'Welcome to the slick ' + chalk.red('Capital Framework component') + ' generator!'
+    ));
+
+    var prompts = [{
+      type: 'input',
+      name: 'name',
+      message: 'What is this project called?',
+      default: this.appname // Default to current folder name
+    }];
+
+    this.prompt(prompts, function (props) {
+      this.props = props;
+      // To access props later use this.props.someOption;
+
+      done();
+    }.bind(this));
+  },
+
+  writing: {
+
+    app: function () {
+      var context = {
+       component_name: this.props.name
```

Comment:
I did it this way since it's in the template file and camelCasing in HTML feels weird... But yeah, it definitely could be.